apps/gitness/src/pages-v2/repo/repo-commits.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from 'react'
+
+import { parseAsInteger, useQueryState } from 'nuqs'
+
+import { useFindRepositoryQuery, useListBranchesQuery, useListCommitsQuery } from '@harnessio/code-service-client'
+import { RepoCommitsView } from '@harnessio/ui/views'
+
+import { useGetRepoRef } from '../../framework/hooks/useGetRepoPath'
+import { PageResponseHeader } from '../../types'
+import { normalizeGitRef } from '../../utils/git-utils'
+
+const sortOptions = [{ name: 'Sort option 1' }, { name: 'Sort option 2' }, { name: 'Sort option 3' }]
```

Comment:
let's not have hardcoded stuff. if api doesn't support sorting, let's just remove it.